README.md

```diff
@@ -71,6 +76,21 @@ envsubst < helm/admin/workflow-pvc.yaml | kubectl apply -n "$NAMESPACE" -f -
 envsubst < helm/admin/secrets.yaml | kubectl apply -n "$NAMESPACE" -f -
 ```
 
+#### Using skaffold
+
+[Skaffold](https://skaffold.dev) can be used to install the OGDC and watch for
+changes in a local environment. Use the `run-local.sh` script to use skaffold:
+
+```
+./scripts/run-local.sh
+```
+
+This will build and deploy the stack to rancher desktop and watch the
+ogdc-runner source for changes. If changes are made, the stack will be rebuilt
+and redeployed to rancher desktop.
+
+#### Manual installation
```

Comment:
I don't think the manual instructions are really necessary to keep around. Let me know if you are OK with removing, otherwise I'll leave as an option! Based on my experience so far, skaffold is a really nice tool to use and should simplify/speed up our local development workflow.

If we like it well enough, it's possible to define separate skaffold profiles for e.g., `local`, `dev`. I figure for now we can continue to manually push changes to our dev cluster, but next time we decide to do this it might be worth revisiting.

Reply:
c6e71b7